common/lib/xmodule/xmodule/modulestore/tests/test_publish.py

```diff
@@ -107,7 +107,7 @@ def test_publish_draft_delete(self):
         if mongo_uses_error_check(self.draft_mongo):
             max_find = 23
         else:
-            max_find = 22
+            max_find = 28
```

Comment:
Why are all of these going up? I thought that this PR was supposed to speed up mongo?